homeassistant/components/sensor/dht.py

```diff
@@ -45,6 +46,10 @@
     vol.Optional(CONF_MONITORED_CONDITIONS, default=[]):
         vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_TEMPERATURE_OFFSET, default=0):
+        vol.All(vol.Coerce(float), vol.Range(min=-500, max=1000)),
+    vol.Optional(CONF_HUMIDITY_OFFSET, default=0):
+        vol.All(vol.Coerce(float), vol.Range(min=-100, max=100))
```

Comment:
Here too. If you need to do wired calculation then a template sensor is the better place.